### PR TITLE
Fix buildifier 0.22.0 lint warnings

### PR DESCRIPTION
**Pull request by smukherj1**
Disabled buildifier in buildkite and enabled it in e2e tests with
specific warnings disabled.

### CODE REVIEW

File: .bazelci/presubmit.yml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 ---
-buildifier: true
```
Comment (nlopezgi) [inline .bazelci/presubmit.yml]: Leave a TODO to turn on again once we can disable specific warnings

File: test-e2e.sh [hunk 2]
```diff
@@ -81,6 +81,18 @@ kubectl version
...
 
+# Install buildifier 0.22.0 and run lint checks
+wget -q https://github.com/bazelbuild/buildtools/releases/download/0.22.0/buildifier
+chmod +x ./buildifier
+
+# Check for issues with the format of our bazel config files.
+# All warnings from https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md
+# are enabled except:
+# rule-impl-return, uninitialized, return-value
+# TODO (suvanjan): reenable once issues are fixed
```
Comment (nlopezgi) [inline test-e2e.sh]: reenable and move to buildkite...

File: .bazelci/presubmit.yml (hunk 1 above)
Comment (smukherj1) [reply .bazelci/presubmit.yml]: Done

File: test-e2e.sh (hunk 2 above)
Comment (smukherj1) [reply test-e2e.sh]: Done

**Pull request merged**